tracing-benchmarks/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ */
+
+buildscript {
+    repositories {
+        maven {
+            url 'https://dl.bintray.com/palantir/releases/'
+        }
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
+    }
+    dependencies {
+        classpath "me.champeau.gradle:jmh-gradle-plugin:0.4.7"
+    }
+}
+
+apply plugin: 'me.champeau.gradle.jmh'
```

Comment:
Can replace the combination of the buildscript block and this apply plugin line with this, since we're pulling from plugins.gradle.org:

```
plugins {
  id "me.champeau.gradle.jmh" version "0.4.7"
}

Reply:
Also I'd quite like to keep the convention of _declaring_ all plugin versions in the root level build.gradle, so there's just one place for people to look to see what functionality a project uses